crates/key-value-store/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+authors = ["Simon Broeng Jensen <sbj@cwconsult.dk>"]
+description = "Key-Value store for LLDAP"
+name = "lldap_key_value_store"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+seaorm = ["dep:sea-orm"]
```

Comment:
This is, I guess, to be able to thread the error to the frontend without depending on sea-orm?

Reply:
It was to get around, that I cant define From<> implementations in another crate than the one that defines the type, and I though it would be nice with some From-impls for sea-orms TransactionError and DbErr, to the error enum. So the are covered behind that feture.